production.ini

```diff
@@ -411,6 +411,10 @@ session.data_dir = %(here)s/data/sessions/data
 session.lock_dir = %(here)s/data/sessions/lock
 session.key = mykey
 session.cookie_on_exception = true
+# Tell the browser to only send the cookie over TLS
+session.secure = true
```

Comment:
You may want to specify that this should be False for local development and True in prod (behind TLS)